.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_NAME:
+        type: string
+        required: true
+      RELEASE_BRANCH:
+        type: string
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: true
+      - run: |
+          error() { echo ">>> $*"; exit 1; }
+
+          npm install -g semver
+          RELEASE_NAME=$(semver ${{ inputs.RELEASE_NAME }})
```

Comment:
Another way I've done this instead of specifying the new version directly, specify a bump level, `patch`, `minor`, or `major`, and use semver to increment the version based on a previous version. It's slightly less manual but interferes with the changelog slightly which looks for a specific version